src/main/java/org/hibernate/annotations/common/reflection/java/generics/ParameterizedTypeImpl.java

```diff
@@ -1,10 +1,8 @@
-/*
- * Hibernate, Relational Persistence for Idiomatic Java
+/* Hibernate, Relational Persistence for Idiomatic Java
```

Comment:
Can we consider dropping this tagline?

I don't see the point of it, and it forces us to configure IDEs differently for each Hibernate project, which is annoying.

Reply:
Ok, right I copied the ASL header from Hibernate Reactive.

How would you have it?

Reply:
```
/*
 * SPDX-License-Identifier: Apache-2.0
 * Copyright: Red Hat Inc. and Hibernate Authors
 */
```

Or (even shorter):

```
// SPDX-License-Identifier: Apache-2.0
// Copyright: Red Hat Inc. and Hibernate Authors
```

Or whatever @sebersole wants to do on ORM. I'm more concerned about using the same header on all projects than about the exact content of the header, TBH.

Reply:
Ok, sure we can do that.

What do other things, let's have a minimalistic header?

Incidentally, headers aren't even mandatory when choosing ASL2 - just a minor preference to make sure contributors are reminded about it once again - but not being essential, if people feel strongly we'd want to be minimalistic, we can also remove them altogether I believe. (I'd need to double check as opinions vary over time.. it was like this last time we discussed it.)

Reply:
Resolved as per decision on https://hibernate.zulipchat.com/#narrow/stream/132094-hibernate-orm-dev/topic/Preferred.20header